netbox/netbox/settings.py

```diff
@@ -124,7 +124,9 @@
 METRICS_ENABLED = getattr(configuration, 'METRICS_ENABLED', False)
 PLUGINS = getattr(configuration, 'PLUGINS', [])
 PLUGINS_CONFIG = getattr(configuration, 'PLUGINS_CONFIG', {})
-QUEUE_MAPPINGS = getattr(configuration, 'QUEUE_MAPPINGS', {})
+QUEUE_MAPPINGS = getattr(configuration, 'QUEUE_MAPPINGS', {
+    None: 'low',
+})
```

Comment:
We should leave this as-is, because it would effect a change to the default configuration, which would require a separate FR. I assume this is for system jobs? If so, we could instead explicitly queue the job in `RQ_QUEUE_LOW`.

Reply:
I've reverted b17b2050df953427769ab6a337c6995ac2a60615. For specifying a custom queue I would not hardcode the low queue, but wait for #16516 to be resolved. As all enqueue methods of `BackgroundJob` pass through `kwargs`, a future `queue` parameter (as suggested in this FR) will be passed through as well.